.mailmap

```diff
@@ -9,6 +9,7 @@
 Aaron Bassett <abassett@gmail.com>
 Abhishek Lekshmanan <abhishek.lekshmanan@ril.com> <abhishek.lekshmanan@gmail.com>
```

Comment:
I'm not sure, without removing it also, if do the credits.sh script from prev release I'm getting the correct output?